hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/metrics/TestNameNodeMetrics.java

```diff
@@ -1128,4 +1131,13 @@ public void testEditLogTailing() throws Exception {
     }
 
   }
+  @Test
```

Comment:
Can you add a line break above this to separate the two test methods?

Reply:
Sure.